INSTALL.md

```diff
@@ -19,6 +19,7 @@ source .venv/bin/activate
 Once you're in your favored virtual environment, install the project dependencies:
 
 ```bash
+python -m pip install --upgrade pip
```

Comment:
I wanted to leave this out, cus some packages still don't fully support the new install conventions. so why push for something that isn't even the default yet, when we're pinned to a python (major) version that came out Oct. 5, 2020?

Reply:
Which packages don't like upgrading pip? Keep in mind that this is the latest version of pip _given_ the python version. I believe it is mostly updating package version collisions & dependency lists.